src/database.py

```diff
@@ -0,0 +1,105 @@
+import sqlite3
+from datetime import datetime
+
+class Database:
+    def __init__(self, db_path='data/database/database.db'):
+        self.conn = sqlite3.connect(db_path)
+        self.cursor = self.conn.cursor()
+        self.create_tables()
+
+    def create_tables(self):
+        self.cursor.execute('''
+            CREATE TABLE IF NOT EXISTS interactions (
+                id INTEGER PRIMARY KEY,
+                npc_name TEXT,
+                interaction_time TIMESTAMP
+            )
+        ''')
+        self.cursor.execute('''
+            CREATE TABLE IF NOT EXISTS map_switches (
+                id INTEGER PRIMARY KEY,
+                from_map TEXT,
+                to_map TEXT,
+                switch_time TIMESTAMP
+            )
+        ''')
+        self.cursor.execute('''
+            CREATE TABLE IF NOT EXISTS goggles_wearing (
+                id INTEGER PRIMARY KEY,
```

Comment:
I'd specify this as `AUTOINCREMENT`, so you don't have to specify the ID manually. In fact, consider this comment true for all occurrences of `INTEGER PRIMARY KEY`

Reply:
if you look closer log methods you will see you don't need to specify ID, ids are already auto incremented